From 485021b1e35c716abaa6c9d85ce501275156a278 Mon Sep 17 00:00:00 2001
From: "Travis A. Everett" <travis.a.everett@gmail.com>
Date: Tue, 31 Mar 2020 17:24:40 -0500
Subject: [PATCH] add barebones setup.py and MANIFEST.in

---
 setup.py | 25 ++++++++++++++++++++++++
 1 file changed, 25 insertions(+)
 create mode 100644 setup.py

diff --git a/setup.py b/setup.py
new file mode 100644
index 00000000..ed7189af
--- /dev/null
+++ b/setup.py
@@ -0,0 +1,25 @@
+#!/usr/bin/env python2
+from setuptools import setup, find_packages, Extension
+
+posix_ = Extension("posix_", sources=["native/posixmodule.c"])
+line_input = Extension(
+    "line_input",
+    sources=["native/line_input.c"],
+    undef_macros=["NDEBUG"],
+    libraries=["readline"],
+)
+fastlex = Extension("fastlex", sources=["native/fastlex.c"], undef_macros=["NDEBUG"])
+libc = Extension("libc", sources=["native/libc.c"], undef_macros=["NDEBUG"])
+
+setup(
+    name="oil",
+    version="0.8",
+    description="A new unix shell",
+    package_dir={"oil": ""},
+    packages=["oil"] + ["oil."+x for x in find_packages()],
+    include_package_data=True,
+    # For posix_methods.def
+    include_dirs=["build/oil-defs", "_devbuild/gen"],  # posix_ & fastlex
+    ext_modules=[posix_, line_input, fastlex, libc],
+    scripts=["bin/oil.py", "bin/osh_eval.py", "bin/osh_parse.py"],
+)
--
2.25.1

