From 64e9a3bd00f762f4f96a9ee74f8c193020f76883 Mon Sep 17 00:00:00 2001
From: "Travis A. Everett" <travis.a.everett@gmail.com>
Date: Fri, 26 Feb 2021 20:04:16 -0600
Subject: revert locale break on macOS

---
 native/libc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/native/libc.c b/native/libc.c
index 2ad3491c..02974c46 100644
--- a/native/libc.c
+++ b/native/libc.c
@@ -22,7 +22,7 @@
 #include <Python.h>
 
 // Oil always uses UTF-8.
-const char* kLocaleOverride = "C.UTF-8";
+const char* kLocaleOverride = "en_US.UTF-8";
 
 // Log messages to stderr.
 static void debug(const char* fmt, ...) {
@@ -346,7 +346,7 @@ func_wcswidth(PyObject *self, PyObject *args){
 
     const char *old_locale = setlocale(LC_CTYPE, NULL);
     if (setlocale(LC_CTYPE, kLocaleOverride) == NULL) {
-        PyErr_SetString(PyExc_SystemError, "C.UTF-8 is not a valid locale");
+        PyErr_SetString(PyExc_SystemError, "en_US.UTF-8 is not a valid locale");
         return NULL;
     }
     int len = mbstowcs(NULL, string, 0);
-- 
2.30.0

